spec/fixtures/complete_case_log.json

```diff
@@ -57,13 +60,13 @@
     "la": "Barnet",
     "property_postcode": "NW1 5TY",
     "property_relet": "No",
-    "rsnvac": "Relet - tenant abandoned property",
+    "rsnvac": "First let of newbuild property",
     "property_reference": "P9876",
     "unittype_gn": "House",
     "property_building_type": "dummy",
     "beds": 3,
     "property_void_date": "03/11/2019",
-    "majorrepairs": "Yes",
+    "majorrepairs": "No",
```

Comment:
How come this one needs to change?

Reply:
Fails if not changed looking at it.
 1) Form Features Soft Validation given a weekly net income that is above the expected amount for the given economic status but below the hard max clears the confirmation question if the amount was amended and the page is returned to using the back button
     Failure/Error: @case_log = CaseLog.find(params[:case_log_id])
     
     ActiveRecord::RecordNotFound:
       Couldn't find CaseLog with 'id'=50 [WHERE "case_logs"."discarded_at" IS NULL]
     # ./app/controllers/case_logs_controller.rb:95:in `block (2 levels) in <class:CaseLogsController>'
     # ------------------
     # --- Caused by: ---
     # Capybara::CapybaraError:
     #   Your application server raised an error - It has been raised in your test code because Capybara.raise_server_errors == true